charts/featureform/charts/api-server/values.yaml

```diff
@@ -43,7 +43,7 @@ service:
   port: 7878
 
 apiserver:
-  port: 7878
+  port: "7878"
```

Comment:
why r there quotes here and not around the other 7878

Reply:
leftover from how I was doing it before. Will fix

Reply:
Fixed